app/grandchallenge/cases/serializers.py

```diff
@@ -99,22 +103,29 @@ class RawImageUploadSessionPatchSerializer(RawImageUploadSessionSerializer):
     reader_study = SlugRelatedField(
         slug_field="slug", queryset=ReaderStudy.objects.all(), required=False
     )
+    answer = PrimaryKeyRelatedField(
+        queryset=Answer.objects.all(), required=False
+    )
 
     class Meta(RawImageUploadSessionSerializer.Meta):
         fields = (
             *RawImageUploadSessionSerializer.Meta.fields,
             "algorithm",
             "archive",
             "reader_study",
+            "answer",
         )
 
     def validate(self, attrs):
         if (
-            sum(f in attrs for f in ["algorithm", "archive", "reader_study"])
+            sum(
+                f in attrs
+                for f in ["algorithm", "archive", "reader_study", "answer"]
+            )
             != 1
         ):
             raise ValidationError(
-                "1 of algorithm, archive or reader study must be set"
+                "1 of algorithm, archive, answer or reader study must be set"
```

Comment:
We also need a validator to ensure that the user has permission to add the image to the answer, see `validate_reader_study` below.